src/components/vulnerability-list/style.css

```diff
@@ -1,34 +1,48 @@
+.severity-list__item {
+  margin-left: var(--eds_spacing_small);
+  margin-right: var(--eds_spacing_small);
```

Comment:
Can be achieved with one line. 
```suggestion
  margin: 0 var(--eds_spacing_small);
```